src/OpenLoco/Vehicles/Vehicle.h

```diff
@@ -157,6 +158,10 @@ namespace OpenLoco::Vehicles
     };
     static_assert(sizeof(TrackAndDirection) == 2);
 
+    // TODO move to a different header
```

Comment:
is this still todo?

Reply:
Yes

Reply:
It will change a good few parts of the code moving this so i don't want to touch it in this pr.

Reply:
no worries, just wanted to check